platform/windows/godot_windows.cpp

```diff
@@ -34,6 +34,16 @@
 #include <locale.h>
 #include <stdio.h>
 
+#ifdef LIVEPP_PATH
+#define STRINGIFY_L(x) STRINGIFY_L_(x)
+#define STRINGIFY_L_(x) L#x
+#define STRINGIFY(x) STRINGIFY_(x)
+#define STRINGIFY_(x) #x
+
+#include STRINGIFY(LIVEPP_PATH/API/LPP_API.h)
+HMODULE livePP;
+#endif
+
```

Comment:
That's a strange way to add a header, why not just use `CPPPATH` in the `detect.py`? There's also `_MKSTR(x)` macro already defined, so no need for custom `STRINGIFY`.

Reply:
Ugh, I haven't thought about `CPPPATH`. That will indeed make it trivial. I'll amend the PR accordingly and get rid of the terribly `STRINGIFY`.